rsts/deployment/plugins/webapi/databricks.rst

```diff
@@ -381,3 +381,54 @@ Wait for the upgrade to complete. You can check the status of the deployment pod
   Make sure you enable `custom containers 
   <https://docs.databricks.com/administration-guide/clusters/container-services.html>`__
   on your Databricks cluster before you trigger the workflow.
+
+Databricks Agent Service configuration
+--------------------------------------
+
+You can use Spark via the Databricks Agent Service.
+
+To do so, follow these steps:
+
+1. Start the agent service using PyFlyte.
+2. Configure the Databricks Agent Service.
+3. Create a secret with the group "databricks" and the key "token."
+    
+   For more details, you can refer to
+   `here <https://docs.flyte.org/projects/cookbook/en/stable/auto_examples/containerization/use_secrets.html>`__.
+
+.. code-block::
+
+  pyflyte serve
+
+.. code-block:: yaml
+
+  tasks:
+    task-plugins:
+      enabled-plugins:
+        - agent-service
+      default-for-task-types:
+        - spark: agent-service
+  plugins:
```

Comment:
Why don't just add it to [here](https://github.com/flyteorg/flyte/blob/9beab93a64ecaf4c97da39e6ca928275036b33d1/rsts/deployment/plugins/webapi/databricks.rst?plain=1#L150)?

Reply:
Because I think it will confuse the user.